src/Components/CreateImageWizardV2/steps/Review/ReviewStepTextLists.tsx

```diff
@@ -0,0 +1,673 @@
+      </TextContent>
+      {isError && (
+        <Alert
+          title="Information about the activation key unavailable"
+          variant="danger"
+          isPlain
+          isInline
+        >
+          Information about the activation key cannot be loaded. Please check
+          the key was not removed and try again later.
+        </Alert>
+      )}
+    </>
+  );
+};
+
+export const ImageDetailsList = () => {
+  const blueprintName = useAppSelector((state) => selectBlueprintName(state));
+  const blueprintDescription = useAppSelector((state) =>
+    selectBlueprintDescription(state)
+  );
+
+  return (
+    <TextContent>
+      <TextList component={TextListVariants.dl}>
+        {blueprintName && (
+          <>
+            <TextListItem
+              component={TextListItemVariants.dt}
+              className="pf-u-min-width"
+            >
+              Image name
+            </TextListItem>
+            <TextListItem component={TextListItemVariants.dd}>
+              {blueprintName}
+            </TextListItem>
+          </>
+        )}
+        {blueprintDescription && (
+          <>
+            <TextListItem
+              component={TextListItemVariants.dt}
+              className="pf-u-min-width"
+            >
+              Description
+            </TextListItem>
+            <TextListItem component={TextListItemVariants.dd}>
+              {blueprintDescription}
+            </TextListItem>
+          </>
+        )}
+      </TextList>
+      <br />
+    </TextContent>
+  );
+};
+{
+  /*
+export const OscapList = () => {
+  const { getState } = useFormApi();
+  const oscapProfile = getState()?.values?.['oscap-profile'];
+  const { data } = useGetOscapCustomizationsQuery({
+    distribution: getState()?.values?.['release'],
+    profile: oscapProfile,
+  });
+
+  return (
+    <TextContent>
+      <TextList component={TextListVariants.dl}>
+        <TextListItem
+          component={TextListItemVariants.dt}
+          className="pf-u-min-width"
+        >
+          Profile name:
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {data?.openscap?.profile_name}
+        </TextListItem>
+      </TextList>
+      <TextList component={TextListVariants.dl}>
+        <TextListItem
+          component={TextListItemVariants.dt}
+          className="pf-u-min-width"
+        >
+          Profile description:
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {data?.openscap?.profile_description}
+        </TextListItem>
+      </TextList>
+      <TextList component={TextListVariants.dl}>
+        <TextListItem
+          component={TextListItemVariants.dt}
+          className="pf-u-min-width"
+        >
+          Reference ID:
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {oscapProfile}
+        </TextListItem>
+      </TextList>
+      <br />
+    </TextContent>
+  );
+};*/
```

Comment:
The profile can be found in the store and all of the other information should be fetchable by an API request.